### DISCUSSION TITLE
Do we want to move the Etcd Community Meeting to CNCF Zoom?

### DISCUSSION THREAD

**Discussion by jberkus** [etcd-io/etcd#15826] [General]
DISCUSSION:

The etcd community meeting is currently hosted on Google corporate Google Meet.  We could move it to CNCF Zoom, which would enable non-Googlers to own the meeting, as well as open the possibility of auto-publishing to Youtube.   This would also make the meeting easier to "find" for the rest of the CNCF community.

There's two parts to this, please vote/comment on each:

1. Should we move the actual meeting from Google Meet to CNCF Zoom account?
2. Should we move our videos from the existing Etcd Youtube channel to the general CNCF account?

Explaining (2): currently we publish to the Etcd youtube channel that's existed since CoreOS.   If we wanted to enable meeting auto-publishing, we'd need to start using the CNCF-owned channels instead.  This would lose some existing audience, but would eliminate some work.

**Comment by jmhbnz** [comment]
Thanks for raising this discussion @jberkus, here are my thoughts:

> Should we move the actual meeting from Google Meet to CNCF Zoom account?

I believe yes - a recent tragic event in another project community has reinforced for me that we should have a way to run the meeting that doesn't rely on any one person. The CNCF is the projects neutral home so it does make sense to me that we use their shared infrastructure to run the meeting. The change needs some good communication and lead time though.

---

> Should we move our videos from the existing Etcd Youtube channel to the general CNCF account?

Do we want to move the old videos? Or is this a question of how we continue to publish new videos?

I think there *might* be benefit to the etcd project to publish meetings and other videos on the CNCF youtube channel as there is a large audience there which means our videos could possibly get a much larger audience and could in turn drive more awareness or engagement with the project.  Currently we are getting about 50 views per month with etcd dedicated channel.

The CNCF youtube channel gets a lot of views per month:

![image](https://user-images.githubusercontent.com/39425256/236314329-f1b3c586-35d5-43cb-916d-ced615a131bb.png)

However it's not necessarily the case that those views will make it to the etcd project videos.  A good comparison is the Envoy project which publishes their community meetings to the CNCF youtube, those only get a small number (~100 ish)  views per video: https://www.youtube.com/playlist?list=PLj6h78yzYM2OI2sdYTykc9ncIjd0tUGZ4 

Putting the views aside, there would be a small toil saving for the person running the community meeting if we do adopt CNCF youtube with their auto publishing feature, I will leave it to the maintainers who have been doing this to comment on this.

**Comment by purpleidea** [comment]
Would https://meet.jit.si/ be a viable option? I've been using it and it works great!

**Comment by ahrtr** [comment]
> 1. Should we move the actual meeting from Google Meet to CNCF Zoom account?
> 2. Should we move our videos from the existing Etcd Youtube channel to the general CNCF account?

Both make sense to me. Thanks @jberkus

**Comment by spzala** [comment]
+1 to both! Thanks @jberkus

**Comment by pchan** [comment]
+1 to both.

**Comment by jberkus** [comment]
@serathius ?

**Reply by serathius** [reply]
+1 to both. We should increase our bus factor.

**Comment by jberkus** [comment]
OK, this seems to be universally approved.  I'll work with CNCF staff to make it happen, once OSS-NA is over.

**Reply by jmhbnz** [reply]
Thanks @jberkus! - Let me know if you need help with anything.

**Reply by ptabor** [reply]
Great. +1 from me.

**Reply by ahrtr** [reply]
Thanks again. +1

**Reply by dims** [reply]
+1 from me!
